BondageClub/Scripts/Speech.js

```diff
@@ -528,11 +557,11 @@ function SpeechBabyTalk(C, CD) {
 			var H = CD.charAt(L).toLowerCase();
 			if (H == "(") Par = true;
 			if (!Par) {
-				if (H == "k" || H == "l") NS = NS + "w";
-				if (H == "s") NS = NS + "sh";
-				if (H == "t") NS = NS + "st";
-				if (H == "a" || H == "b" || H == "c" || H == "d" || H == "e" || H == "f" || H == "g" || H == "h" || H == "i" || H == "j" || H == "m" || H == "n" || H == "o" || H == "p" || H == "q" || H == "r" || H == "u" || H == "v" || H == "w" || H == "x" || H == "y" || H == "z" || H == " " || H == "'" || H == "?" || H == "!" || H == "." || H == ",") NS = NS + H;
-			} else NS = NS + CD.charAt(L);
+				if (dialogContainsCharacter('kl', H)) NS += 'w';
+				if (H == "s") NS = NS += 'sh';
+				if (H == "t") NS = NS += 'st';
+				if (H.match('[a-z]') || dialogContainsCharacter(" '?!.,", H)) NS += H;
```

Comment:
Honestly... why not just use regexes for everything? It would look much cleaner and you don't need any extra functions...
```suggestion
				if (/[a-z '?!.,]/.test(H)) NS += H;
```

Reply:
But definitely let me know what you think. In general, If we want to replace all of the if statements that using includes with the regex, as stated in the previous comment, the string that we want to look for is not always be consistent, that means, regex needs to be compiled multiple times during a chat room session. This may cause a significant performance impact especially for older PCs.

Reply:
Sorry I think I mis-read the comment lol. Nice catch.